roles/pulp3-content/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Install pulp_content_app service file
+  template:
+    src: pulp_content_app.service.j2
+    dest: /lib/systemd/system/pulp_content_app.service
+    owner: root
+    group: root
+    mode: 0644
+  notify: Reload systemd and restart pulp content app
+  become: true
```

Comment:
both of these tasks should move into a block with `become: true` since they need to be run as the same user in all cases.